rfc/5280/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "asn1.js-rfc5280",
+  "version": "0.0.1",
+  "description": "RFC5280 extension structures for asn1.js",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:indutny/asn1.js"
+  },
+  "keywords": [
+    "asn1",
+    "rfc3280",
+    "der"
+  ],
+  "author": "Fedor Indutny",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/indutny/asn1.js/issues"
+  },
+  "homepage": "https://github.com/indutny/asn1.js",
+  "peerDependencies": {
+    "asn1.js": "~0.4.0"
```

Comment:
Huh, recent `asn1.js` is `4.4.0`. Let's make it a normal dependency too: `^4.4.0`